apps/meet/src/components/SelectPersonView.tsx

```diff
@@ -38,32 +40,17 @@ const SelectPersonView: React.FC<SelectPersonViewProps> = ({ page: { groupId },
     );
   }
 
-  if (data.type === 'redirect') {
```

Comment:
This was dead code as the 'redirect' case was no longer present in the API route